docs/docs/install/install.md

```diff
@@ -70,6 +70,12 @@ Configuration can be done by both environment variable(conf/zeppelin-env.sh) and
     <td>Zeppelin server port. Note that port+1 is used for web socket</td>
   </tr>
   <tr>
+    <td>ZEPPELIN_CONTEXT_PATH</td>
```

Comment:
could this be `ZEPPELIN_SERVER_CONTEXT_PATH`?
there is a object called ZeppelinContext in the Spark interpreter group and this name might be confusing